subprojects/base-services/src/main/java/org/gradle/internal/operations/OperationIdentifier.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.internal.operations;
+
+public class OperationIdentifier {
```

Comment:
As silly as it might sounds but let's make sure instances can be compared by adding a test e.g. similar to `DefaultModuleComponentIdentifierTest`.